shaders/src/main/glsl/samples/320es/stable_mergesort.frag

```diff
@@ -25,6 +25,25 @@ uniform vec2 injectionSwitch;
 
 uniform vec2 resolution;
 
+/*
+This shader populates data array with reversed values,
+performs merge sort to get the data to the correct order,
+and then draws an image based on the sorted values and
+the screen coordinates.
+
+The sorting operation is purely integer math, and the
```

Comment:
"is purely integer math" -> "uses purely integer math"

Reply:
done